tools/mafft/mafft-add.xml

```diff
@@ -0,0 +1,88 @@
+    <exit_code range=":-1" level="fatal" description="Error occurred. Please check Tool Standard Error" />
+  </stdio>
+  <version_command>    <![CDATA[
+    mafft --version
+    ]]>
+  </version_command>
+  <command>
+    <![CDATA[
+      mafft
+      #if $sequences.sequenceType == 'singleseq'
+        $sequences.preservegap  $inputSequences
+      #elif $sequences.sequenceType == 'frags'
+        --addfragments $inputSequences
+      #elif $sequences.sequenceType == 'group'
+        --addprofile $inputSequences
+      #end if
+      $keeplength $map $reorder
+      $inputAlignment
+      > $outputAlignment
+      #if $map == '--mapout'
+        && mv ${inputSequences}.map $outputmap
+      #end if
+    ]]>
+  </command>
+  <inputs>
+    <param name="inputSequences" type="data" format="fasta" label="Sequences to add to the alignment" help="Amino acid or nucleotide sequences in FASTA format."/>
+    <param name="inputAlignment" type="data" format="fasta" label="Alignment" help="Amino acid or nucleotide sequences in aligned FASTA format."/>
+    <conditional name="sequences">
+      <param name="sequenceType" type="select" label="What do you want to add to the alignment" >
+        <option value="singleseq">A single sequence</option>
+        <option value="frags" selected="true">Fragments</option>
+        <option value="group">An alignment</option>
+      </param>
+      <when value='singleseq'>
+        <param name="preservegap" type="select" label="Preserve the original alignment" help="Keep the given alignment unchanged (--add .)If not, the aligned letters in the seed alignment are preserved but gaps are not necessarily preserved (--seed)">
+          <option value="--add" selected="true" >Yes</option>
+          <option value="--seed">no</option>
+        </param>
+      </when>
+      <when value='frags'/>
+      <when value='group'/>
+    </conditional>
+    <param name="keeplength" type="boolean" truevalue="--keeplength" falsevalue="" checked="True" label="Keep alignment length" help="The alignment length is unchanged.  Insertions at the additional sequences are deleted" />
+    <param name="map" type="boolean" truevalue="--mapout" falsevalue="" checked="False" label="Output a correspondance table of positions (--mapout)" help="Output a correspondence table of positions, sequence.map, between before and after the calculation.  The --mapout option automatically turns on the --keeplength option, to keep the numbering of sites in the reference alignment" />
+    <param name="reorder" type="boolean" truevalue="" falsevalue="--reorder" checked="True" label="Preserve the original order of sequences (--reorder)"  />
+  </inputs>
+  <outputs>
+    <data format="fasta" name="outputAlignment" label="${tool.name} on ${on_string} : New alignment"/>
+    <data name="outputmap" format="tabular" label="${tool.name} on ${on_string} : Correspondance of positions ">
+      <filter>map == True</filter>
+    </data>
+  </outputs>
+  <tests>
+    <test>
```

Comment:
can you add here the expected_num_outputs ... https://docs.galaxyproject.org/en/latest/dev/schema.html#id78